SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    env = !echo $CONDA_PREFIX
</pre><p>+1 to using os.getenv here.</p><p> </p><p>If we're adding this code to multiple notebooks, it seems like we should add this as a function in our library.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>